Shared/Supporting Files/Views/AboutView.swift

```diff
@@ -58,7 +58,7 @@ private struct VersionRow: View {
 }
 
 private extension Bundle {
-    static let arcGIS = Bundle(identifier: "ArcGIS")!
+    static let arcGIS = Bundle(identifier: "com.esri.ArcGIS")!
```

Comment:
This line will crash if you use the local package, it is expected to work with the GitHub repo.